tests/data_context/test_data_context_ge_cloud_mode.py

```diff
@@ -46,17 +53,14 @@ def test_data_context_ge_cloud_mode_makes_successful_request_to_cloud_api(
     ):  # Not concerned with constructor output (only evaluating interaction with requests during __init__)
         pass
 
-    called_with_url = f"{ge_cloud_runtime_base_url}/organizations/{ge_cloud_runtime_organization_id}/data-context-configuration"
-    called_with_header = {"headers": request_headers}
-
     # Only ever called once with the endpoint URL and auth token as args
-    mock_request.assert_called_once()
-    assert mock_request.call_args[0][0] == called_with_url
-    assert mock_request.call_args[1] == called_with_header
+    assert responses.assert_call_count(called_with_url, 1) is True
 
 
+@responses.activate
+@pytest.mark.unit
 @pytest.mark.cloud
-@mock.patch("requests.get")
+@mock.patch("requests.Session.get")
```

Comment:
I'd like to make these all use a `responses` mock, but in the interest of getting this merged quickly, I'm leaving these like this.